rfc/008-platform-support.md

```diff
@@ -0,0 +1,111 @@
+# RFC 008: Platform Support
+
+## Objective
+
+Implement support for diverse platforms in Contrast, ensuring that:
+
+1. Users can mix-and-match features throughout all platforms supported by Contrast
```

Comment:
I don't like how this rfc mixes a user facing feature with a refactoring. For an rfc for a feature, I'd expect far more arguing from the users perspective and UX considerations.

Reply:
It was meant to propose only a refactoring of internal code first, but now shifted towards something that feels like a proposal for a configuration file instead. Let's discuss this in a meeting, and I'll then re-write the RFC to argue more from a user's perspective.